.github/workflows/workflow-vercel-deployment.yml

```diff
@@ -1,11 +1,22 @@
 name: Vercel Preview Deployment
 
+on:
+  workflow_call:
+    inputs:
+      vercel-project-name:
```

Comment:
as discussed in the ADR, we shouldn't exposed implementation specific naming. 

could we just call this project-name as a parameter?

Reply:
yeah sounds good I changed it